### ISSUE TITLE
[Feature request] Separate taskbar into separate dock

### ISSUE THREAD

**Issue by PohaoGit** [home-sweet-gnome/dash-to-panel#457]
Thank you for making this awesome extension. Everything works fine, I like how dash-to-panel handles the app launcher. But I want to separate the system tray and the app launcher. I have some many extensions installed and they have icons on the system tray. If the system tray and the app launcher are integrated into one panel that's way too small to display them all. So I hope we can have an option to leave the system tray separated.

**Comment by zemelax** [comment]
agree 
does not show the tray icon (telegram, rocket chat and other) is not very convenient. Please correct

**Comment by jderose9** [comment]
Hello! Dash-to-panel actually does not integrate the system tray icons into the panel. If they are integrated and you do not want them, you likely have an extension installed such as AppIndicator or TopIcons Plus.

@zemelax I think you may have the opposite issue of wanting to see the icons, in which case you would want to install one of these other extensions.

**Comment by PohaoGit** [comment]
@jderose9 Actually, it's not just icons. I use other extensions to show such as CPU and RAM info, title bar, weather, etc on the system tray. So I hope I can keep all this information with the system tray on the top, and dash-to-panel on the bottom.

**Comment by jderose9** [comment]
@PohaoGit ah ok, I believe you are looking for the extension Dash-to-Dock then.

**Comment by PohaoGit** [comment]
@jderose9 Not really, dash-to-panel is much more powerful than dash-to-dock , I can't use Gnome without it. Ungroup icons, change icon paddings, behavior when clicking on icons, etc.

**Comment by purplexss** [comment]
+1 with this feature request

thanks again for this useful extension team :)

**Comment by lunetics** [comment]
+1 Would also love this.

**Comment by corrilan** [comment]
+1 with this feature request

Thanks for all your hard work so far.

**Comment by jasiralavi** [comment]
Wouldn't it be better if we could move the icons that are not frequently used to a tray like in windows? So it's still there and we don't need another dock. 

Please see below screenshots.

![image](https://user-images.githubusercontent.com/16372030/69121369-32c62100-0ac2-11ea-8884-2567f2571276.png)
![image](https://user-images.githubusercontent.com/16372030/69121394-44a7c400-0ac2-11ea-921d-b30faf6d4f01.png)

Images from: https://is.gd/p3BALX

**Comment by zemelax** [comment]
yea, its wold be great

**Comment by ipsingh06** [comment]
+1 I would like to use just the opened windows part of the panel as Window List replacement. It'd be great if you are able to keep the original top panel and create a new panel for Dash To Panel.

**Comment by levacic** [comment]
I'd actually like the same thing as what @ipsingh06 originally asked about - the default top bar (for various extensions such as a system monitor etc.) and Dash to panel on the bottom as a powerful replacement for Window list (which doesn't have the option of rearranging programs).

The scope of the feature seems like it could be a few configuration options:

1. Add a "Hidden" option for the "Clock location" setting 
2. Add a "Display the status menu on the main panel" toggle switch (similar to the option for multiple monitors) 
3. Add a "Display original Ubuntu Dock" option 

It's easy to manually hide the actual dock of the bundled Ubuntu dock and leave the top bar/panel visible by toggling a few options via dconf - these are `autohide`, `dock-fixed` and `intellihide` in `/org/gnome/shell/extensions/dash-to-dock` - so this extension wouldn't _have_ to deal with those either. 

Any tips on where to start with these if someone wanted to submit a PR? They seem like easy-to-implement features, except for maybe number 3, in case Dash to panel somehow internally relies on the original dock being disabled.

The other idea of having multiple Dash to panel instances (e.g. top and bottom) seems like it would be a separate feature request.

**Comment by charlesg99** [comment]
Hello guys, an option to keep the stock gnome-shell top panel has been added to the master branch. I do get a definite Ubuntu feel, but I like it a lot more that I thought I would! Feel free to install from source to give it a try and report with feedback! Thanks!

![image](https://user-images.githubusercontent.com/33044886/80870725-6662c500-8c76-11ea-859e-3a12bc268896.png)

**Comment by levacic** [comment]
Thanks, that works great with Ubuntu 16.04!

Finally an amazing replacement for the "Window List" extension - my top+bottom panel setup was so important to me that I stuck around with that extension for years - I was so used to this setup from Mint+Cinnamon that I later used it with XFCE and Mate as well - but unfortunately Gnome was the only one that worked flawlessly with a 4K display as of 16.04 (which is around the time when I got a 4K display) so I got kinda stuck there. As that's what I'm still using (until I can upgrade to 20.04, hopefully soon - which I've tested and indeed it does support my hardware setup), just wanted to report back - so far, so good.

FWIW on Ubuntu 16.04 and Gnome Shell 3.18.5 (in case this is helpful for anyone):

1. I uninstalled the original extension from extensions.gnome.org (via the browser integration)
2. Confirmed it was deleted from `~/.local/share/gnome-shell/extensions`
3. Cloned the repo
4. Ran `make install`
5. Restarted my session
6. Enabled the extension via the Tweak Tool

Everything worked without any issues.

Appreciate your hard work on this extension, I'll write back if I encounter any issues.
